### PR TITLE
[Tabs] Don't use MDCTabBarColorThemer in examples

### PR DESCRIPTION
**Pull request by andrewoverton**
This PR makes it so that Tabs examples use the theming extension instead of the themer.

Related to #9061.
Related to #9063.
Related to #9062.

### CODE REVIEW

File: components/Tabs/examples/BottomNavigationBarExample.m [hunk 1]
```diff
@@ -40,7 +39,7 @@ - (void)viewDidLoad {
   _bottomNavigationBar = [[MDCTabBar alloc] initWithFrame:CGRectZero];
   _bottomNavigationBar.translatesAutoresizingMaskIntoConstraints = NO;
   _bottomNavigationBar.delegate = self;
-  [MDCTabBarColorThemer applySemanticColorScheme:self.colorScheme toTabs:_bottomNavigationBar];
+  [_bottomNavigationBar applyPrimaryThemeWithScheme:_containerScheme];
```
Comment (romoore) [inline components/Tabs/examples/BottomNavigationBarExample.m]: Tip: Please use `self.containerScheme`.

File: components/Tabs/examples/TabBarTextOnlyExample.m [hunk 2]
```diff
@@ -29,8 +29,7 @@ - (id)initWithCollectionViewLayout:(UICollectionViewLayout *)layout {
   self = [super initWithCollectionViewLayout:layout];
   if (self) {
     [self setupExampleViews:@[ @"Change Alignment", @"Toggle Case", @"Clear Selection" ]];
-    self.colorScheme =
-        [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
+    self.containerScheme = [[MDCContainerScheme alloc] init];
```
Comment (romoore) [inline components/Tabs/examples/TabBarTextOnlyExample.m]: Please set the ivar directly.

```_containerScheme = [[MDCContainerScheme alloc] init];```

File: components/Tabs/examples/TabBarInterfaceBuilderExample.m [hunk 3]
```diff
@@ -32,8 +32,7 @@ @implementation TabBarInterfaceBuilderExample
 - (id)init {
   self = [super init];
   if (self) {
-    self.colorScheme =
-        [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
+    self.containerScheme = [[MDCContainerScheme alloc] init];
```
Comment (romoore) [inline components/Tabs/examples/TabBarInterfaceBuilderExample.m]: Please set the ivar directly.

```
_containerScheme = [[MDCContainerScheme alloc] init];
```

File: components/Tabs/examples/TabBarIconExample.m [hunk 4]
```diff
@@ -15,22 +15,21 @@
...
 
 @implementation TabBarIconExample
 
 - (id)init {
   self = [super init];
   if (self) {
-    self.colorScheme =
-        [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
-    self.typographyScheme = [[MDCTypographyScheme alloc] init];
+    self.containerScheme = [[MDCContainerScheme alloc] init];
```
Comment (romoore) [inline components/Tabs/examples/TabBarIconExample.m]: Please assign to the ivar directly:

```
_containerScheme = [[MDCContainerScheme alloc] init];
```

File: components/Tabs/examples/BottomNavigationBarExample.m [hunk 5]
```diff
@@ -28,8 +28,7 @@ @implementation BottomNavigationBarExample {
 - (id)init {
   self = [super init];
   if (self) {
-    self.colorScheme =
-        [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
+    self.containerScheme = [[MDCContainerScheme alloc] init];
```
Comment (romoore) [inline components/Tabs/examples/BottomNavigationBarExample.m]: Please assign the ivar directly:

```
_containerScheme = [[MDCContainerScheme alloc] init];
```

File: components/Tabs/examples/TabBarViewControllerExample.m [hunk 6]
```diff
@@ -25,9 +25,7 @@ @implementation TabBarViewControllerExample
 - (id)init {
   self = [super init];
   if (self) {
-    self.colorScheme =
-        [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
-    self.typographyScheme = [[MDCTypographyScheme alloc] init];
+    self.containerScheme = [[MDCContainerScheme alloc] init];
```
Comment (romoore) [inline components/Tabs/examples/TabBarViewControllerExample.m]: Please set the ivar directly.

File: components/Tabs/examples/supplemental/TabBarViewControllerExampleSupplemental.m [hunk 7]
```diff
@@ -152,8 +152,8 @@ - (void)setupTabBarColors {
 - (nonnull NSArray *)constructExampleViewControllers {
   NSBundle *bundle = [NSBundle bundleForClass:[TabBarViewControllerExample class]];
   MDCButtonScheme *buttonScheme = [[MDCButtonScheme alloc] init];
-  buttonScheme.colorScheme = self.colorScheme;
-  buttonScheme.typographyScheme = self.typographyScheme;
+  buttonScheme.colorScheme = self.containerScheme.colorScheme;
```
Comment (romoore) [inline components/Tabs/examples/supplemental/TabBarViewControllerExampleSupplemental.m:155]: Should the container scheme be initialized in this implementation at some point?  Maybe in `init` or `viewDidLoad` or a similar method?

**Review by romoore — APPROVED** [review]

File: components/Tabs/examples/supplemental/TabBarViewControllerExampleSupplemental.m (hunk 7 above)
Comment (andrewoverton) [reply components/Tabs/examples/supplemental/TabBarViewControllerExampleSupplemental.m:155]: It happens in init but it's in a different file, `TabBarViewControllerExample.m`. These examples are set up kind of weird it seems.

**Pull request merged**